invenio/ext/login/legacy_user.py

```diff
@@ -59,6 +67,20 @@
 }
 
 
+def isUserAdmin(user_info):
```

Comment:
They are called only inside `legacy_user.py` itself and from legacy `bibdocfile`.
I think we can consider it as legacy and remove after all legacy code will be removed.